### ISSUE TITLE
No project ID specified in app.json, unable to sign manifest

### ISSUE THREAD

**Issue by yonitou** [expo/expo#17379]
### Summary

Hi, 

Just upgraded from SDK 44 to SDK 45. 

Everytime I'm launching my app or reloading the JS bundle, I'm having this warning : `No project ID specified in app.json, unable to sign manifest`

<img width="622" alt="Capture d’écran 2022-05-06 à 15 56 50" src="https://user-images.githubusercontent.com/61981305/167147464-98c0b2d5-101a-4bef-8170-c6777211aab9.png">

Never had this warning before and I don't know if this is important or not. I'm also using EAS Updates. 

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Android, iOS

### SDK Version (managed workflow only)

45

### Environment

```
 expo-env-info 1.0.3 environment info:
    System:
      OS: macOS 12.3.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 16.13.1 - ~/.nvm/versions/node/v16.13.1/bin/node
      npm: 8.3.0 - ~/.nvm/versions/node/v16.13.1/bin/npm
    Managers:
      CocoaPods: 1.11.2 - /Users/yonitouboul/.rbenv/shims/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2020.3 AI-203.7717.56.2031.7935034
      Xcode: 13.3.1/13E500a - /usr/bin/xcodebuild
    npmPackages:
      babel-preset-expo: ~9.1.0 => 9.1.0 
      expo: ^45.0.0 => 45.0.0 
      react: 17.0.2 => 17.0.2 
      react-dom: 17.0.2 => 17.0.2 
      react-native: 0.68.1 => 0.68.1 
    npmGlobalPackages:
      eas-cli: 0.52.0
      expo-cli: 5.4.3
    Expo Workflow: managed
```

### Reproducible demo

Upgrade from 44 to 45 and run `expo start`

**Comment by animaonline** [comment]
Just started experiencing the same error after upgrading.

**Comment by animaonline** [comment]
Removing 
`    "runtimeVersion": "1.0.0"`
from app.json seems to help, though I dunno how that will affect expo build...

**Comment by kouta77** [comment]
I have this exact same problem, in the exact same circumstances...but I don't have runtimeVersion and still get the same error

**Comment by yonitou** [comment]
I'm using the runtimeVersion myself to distribute over-the-air updates using Expo Updates so I can't get rid of it.

**Comment by animaonline** [comment]
@yonitou I found a fix. Run: `eas update:configure`

More info: 
https://docs.expo.dev/eas-update/getting-started/

**Comment by kouta77** [comment]
> 
Mmm not sure if this is a real fix, I'm still getting the same error message :(

**Comment by yonitou** [comment]
Thanks for the try @animaonline , but I already checked it when I ran into my issue and it didn't fix it :/ Happy it solved yours !

**Comment by yonitou** [comment]
Is it possible to have an answer or at least, a feedback, from the Expo team ? 
Thanks

**Comment by mister-conor** [comment]
I encountered this issue also. We upgraded to Expo 45 so I ran the command `eas update:configure` to configure updates as we had been previously using `expo publish` for OTA updates. Expo 45 does not support updates for free tier right now, but it will be available later in 2022 according to the [Expo 45 release notes](https://blog.expo.dev/expo-sdk-45-f4e332954a68).

From the link:

**_
Later this year, we’ll release EAS Update to all accounts, allow secure end-to-end code signing, and much more. Stay tuned!_**

I fixed the issue by removing the changes that the command `eas update:configure`  had made to the app.json file. I removed the "updates" and "runtimeVersion" sections in the app.json. I also uninstalled the `expo-updates` package as it is of no use until it is made available to the free tier.

I hope this helps someone else. It is frustrating that Expo had previously made `expo publish` commands available to all users but now the updates are only availavle to paid accounts until later this year. We used the OTA updates quite alot to deploy to testers and also patches to production versions of our app.
